floater/utils.py

```diff
@@ -218,3 +218,57 @@ def floats_to_castra(input_dir, output_fname, progress=False, **kwargs):
         if not c:
             c = Castra(output_fname, template=df)
         c.extend(df)
+
+def floats_to_netcdf(input_dir, output_fname, float_file_prefix):
+    """Convert MITgcm float data to NetCDF format.
+
+    Parameters
+    ----------
+    input_dir : path
+        Where to find the MITgcm output data
+    output_fname : path
+        Filename of the NetCDF data store
+    float_file_prefix: str
+        Prefix of MITgcm output files
+    """
+    import dask.dataframe as dd
+    import xarray as xr
+
+    float_digits = 10
+    float_columns = ['npart', 'time', 'x', 'y', 'z', 'u', 'v', 'vort']
+    float_dtypes = np.dtype([('npart', np.int32), ('time', np.float32), ('x', np.float32), ('y', np.float32), ('z', np.float32), ('u', np.float32), ('v', np.float32), ('vort', np.float32)])
+
+    reference_time = np.datetime64('1993-01-01', 'ns')
+    timestep = 86400.
+
+    float_list = []
+    for file in os.listdir(input_dir):
+        if file.startswith(float_file_prefix) and file.endswith('.csv'):
+            float_list.append(file[:len(float_file_prefix)+float_digits+1])
+            float_list = sorted(list(set(float_list)))
```

Comment:
@geosciz I think this is does what you want:
``` python
from glob import glob

float_files= glob('float_file_prefix' + '*.csv')
float_file_timesteps = {f[:len(float_file_prefix)+float_digits+1] for f in float_files} 
```
You can then `for` loop over the set `float_file_timesteps`.